swarm/src/main/resources/swarm/swarm.bom

```diff
@@ -601,6 +601,17 @@ brooklyn.catalog:
               DOCKER_HOST: $brooklyn:attributeWhenReady("swarm.url")
             command: |
               docker info | grep "^Containers:" | sed -e "s/^Containers: //"
+        - type: org.apache.brooklyn.core.effector.ssh.SshCommandEffector
```

Comment:
Looking at the other `SshCommandSensor`s which run `docker info`, they include the following:

```
            shell.env:
              DOCKER_HOST: $brooklyn:attributeWhenReady("swarm.url")
```

I think you probably need that here as well, for the `docker` ssh command to work.